Rename.java

```diff
@@ -0,0 +1,37 @@
+
+import java.io.File;
+
+/**
+ * class Rename that has a boolean method renameFile 
+ * @author Milena Vallejos 
+ * @date May 24
+ */
+public class Rename
+{ /**
+     * Method to renameFile 

```

Comment:
work more on the description, i know that rename files and what happens if the file already exists? put more detail in the description